### PR TITLE
Fixes broken link in updating docs

### PR DESCRIPTION
**Pull request by opayne1**
Fixing a link in upstream docs that goes to 404 page.

- Specifically the "About the OpenShift Update Service" in the Additional resources section.

Version(s):
okd and enterprise-4.11

Issue:
Discussed in slack with Console Team

Link to docs preview:
OKD
http://file.rdu.redhat.com/opayne/link-fix-updating-restricted/welcome/
Direct link: http://file.rdu.redhat.com/opayne/link-fix-updating-restricted/updating/understanding-openshift-updates.html

Enterprise
https://47272--docspreview.netlify.app/openshift-enterprise/latest/updating/understanding-openshift-updates.html

### CODE REVIEW

**Review by GroceryBoyJr — COMMENTED** [review]
/LGTM!

File: updating/understanding-openshift-updates.adoc [hunk 1]
```diff
@@ -18,5 +18,5 @@ include::modules/update-common-terms.adoc[leveloffset=+1]
 .Additional resources
 
 * xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-overview-post-install-machine-configuration-tasks[Machine config overview]
-* xref:../updating/updating-restricted-network-cluster.adoc#update-service-overview_updating-restricted-network-cluster[About the OpenShift Update Service]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-restricted-network-cluster-update-service[About the OpenShift Update Service]
```
Comment (opayne1) [inline updating/understanding-openshift-updates.adoc:21]:
```suggestion
ifdef::openshift-enterprise[]
* xref:../updating/updating-restricted-network-cluster.adoc#update-restricted-network-cluster-update-service[About the OpenShift Update Service]
 endif:[]
```

**Comment by bergerhoffer** [comment]
The enterprise-4.12 label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.11. And any PR going into `main` must also target the latest version branch (enterprise-4.12).

If the update in your PR does NOT apply to version 4.12 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by maxwelldb** [comment]
@opayne1 Hi! Is this ready for peer review? Just noting the label, but you do have an unresolved suggestion here: https://github.com/openshift/openshift-docs/pull/47272#discussion_r916248850

**Review by Wiharris — COMMENTED** [review]
/LGTM!

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Pull request merged**